integreat_compass/cms/templates/offers/image_picker.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
-<div class="title-image-preview-container relative rounded rounded-3xl overflow-hidden {% if widget.value == default_title_image or not widget.value.url.strip %}hidden{% endif %}">
+<div class="title-image-preview-container relative rounded rounded-3xl overflow-hidden {% if not widget.value.url.strip %}hidden{% endif %}">
```

Comment:
```suggestion
<div class="title-image-preview-container relative rounded rounded-3xl overflow-hidden {% if not widget.value %}hidden{% endif %}">
```
This is probably sufficient?